src/tests/stress/pedsim/Ped_util/Regions.enc

```diff
@@ -0,0 +1,480 @@
+module Regions
+import Agent_passive
+import Global_funs
+import Quad_tree
+
+fun regions(var agents : [Agent], boxes : int) : unit
+  var box = new Tiling_box(agents, boxes)
+  box!move(10000)
+end
+fun link(a : Box, b : Box) : bool
+  val aa = a!link(b)
+  val bb = b!link(a)
+  val aaa = get(aa)
+  val bbb = get(bb)
```

Comment:
you are getting on `bb` but not using the result. Is this for synchronisation purposes?